App2/App2/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите длину ребра октаэдра:");
+        double a = double.Parse(Console.ReadLine());
+
+        double s = 2 * Math.Sqrt(3) * Math.Pow(a, 2);
+        double v = Math.Sqrt(2) * Math.Pow(a, 3) * 1 / 3;
```

Comment:
Зачем такие сложности?
```suggestion
        double v = Math.Sqrt(2) * Math.Pow(a, 3) / 3;
```